### ISSUE TITLE
Why test with exception in test_import?

### ISSUE THREAD

**Issue by DrDomenicoMarson** [alchemistry/alchemlyb#241]
I was checking for coverage and I saw this test 
```python
import alchemlyb 
 
def test_name(): 
    try: 
        assert alchemlyb.__name__ == 'alchemlyb' 
    except Exception as e: 
        raise e 
```
Why are we catching an Exception here in a test?

**Comment by xiki-tempula** [comment]
No idea. It seems to me that `assert alchemlyb.__name__ == 'alchemlyb'`  should be enough. Could you use blame to check who wrote that part?

**Comment by DrDomenicoMarson** [comment]
I didn't know about "blame" :-)

It seems @ianmkenney did the commit 6 years ago! Could we just remove the `try`?

**Comment by ianmkenney** [comment]
I couldn't tell you why I thought that was either necessary or useful. Feel free to change!

**Comment by DrDomenicoMarson** [comment]
Great, I implemented the change in PR #240, I hope it's fine, just not to have another PR just for that!

**Comment by orbeckst** [comment]
If you had done another PR, I would have just merged it.

Micro-PRs are easier to maintain.
